app/src/index.css

```diff
@@ -27,7 +27,7 @@
   --color-white: #ffffff;
   --color-black: #000000;
 
-  --error-color: var(--color-pale);
+  --error-color: var(--color-red);
```

Comment:
```suggestion
  --error-color: var(--color-orange-0);
```

Reply:
I updated it to orange but it's still a bit difficult to read. How about if we add a black background color to the h4 element?

Reply:
Yes that can work, better that it's legible and we can always review later to improve. I think the only two pages with the green background are the signup page and the password recovery so we could just add the styling to those.